src/screens/home/components/articles/index.jsx

```diff
@@ -18,14 +18,26 @@ const Articles = ({subreddit}) => {
             }
         })
 
-    });
+    }));
+
+    }, [subreddit])
 
-    }, [subreddit]);
+    const fetchData = () => {
+        if (articles.after == null) {
+            setArticles(articles.length = articles.lengh + 3)
+        }
```

Comment:
You shouldn't assign some value in array.length, `length` is a dynamic property of an array instance.

Reply:
This method get/fetch the data from the server.
You should implement the something like this here:


!Important: incomplete code below:
```
fetch(`https://www.reddit.com/r/${encodeURI(subreddit)}/new.json?limit=3`).then(res=>{
        if (res.status !=200) {
        console.log('Error');
        return;
...etc
```

After these changes, you should remove the code in useEffect, to avoid duplicates